src/ripple/app/paths/PathRequest.cpp

```diff
@@ -585,11 +585,11 @@ PathRequest::findPaths (std::shared_ptr<RippleLineCache> const& cache,
         {
             Json::Value jvEntry (Json::objectValue);
             rc.actualAmountIn.setIssuer (sourceAccount);
-            jvEntry[jss::source_amount] = rc.actualAmountIn.getJson (0);
-            jvEntry[jss::paths_computed] = ps.getJson(0);
+            jvEntry[jss::source_amount] = rc.actualAmountIn.getJson (JsonOptions::none);
```

Comment:
This line needs to be broken. I'll forego hunting down the rest until you've had a chance to take a first pass finding these newly long lines.